earn/src/pages/MarketsPage.tsx

```diff
@@ -30,17 +30,17 @@ import { fetchBorrowerDatas, MarginAccount } from '../data/MarginAccount';
 import { PriceRelayLatestResponse } from '../data/PriceRelayResponse';
 import { getProminentColor } from '../util/Colors';
 
+const UNSELECTED_OPTION_TEXT_COLOR = 'rgba(75, 105, 128, 1)';
```

Comment:
Where is this coming from? If it's from the spec, we should define it in shared, and if not, let's switch to something that is -- will grey 600 or 700 work?

Reply:
It is the same color used for the navbar

Reply:
Just checked, this is GREY_400. Will add it to shared